js/controller.js

```diff
@@ -269,7 +269,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         element = elementVideo;
```

Comment:
Isn't the point of this event that the element should exist? Do we understand why we're getting an event without the element being instantiated?